contracts/walletCache.sol

```diff
@@ -67,7 +66,6 @@ contract WalletCache is ENSResolvable, Controllable {
         _initializeENSResolvable(_ens_);
         _initializeControllable(_controllerNode_);
         walletImplementation = _walletImplementation_;
-        ens = _ens_;
```

Comment:
_ens_ should also be removed from the contractor argument list.

Reply:
no, we want to pass it in ?